components/Grid/grid.css

```diff
@@ -0,0 +1,62 @@
+
+.grid-container-2{
+    display: grid;
+    grid-gap: 1rem;
+    padding: 1.5rem 1.5rem;
+    box-shadow: 1px 2px 5px 3px rgba(0, 0, 0, 0.1);
+    grid-template-columns: repeat(2, 15rem);
+    justify-content: center;
+    align-items: center;
+}
+
+.grid-img{
+  border: 1px solid;
+  padding: 1.2rem;
+  margin: 1rem;
+  display: flex;
+  color: transparent;
+}
+
+
+.img1{
+    width:10rem;
+    height:10rem;
+}
+
+.img2{
+    width:10rem;
+    height:10rem;
+}
```

Comment:
can give image width as 100% and reduce the container width, might be good for resolution

Reply:
Actually I tried doing it. But it is not working. Images are getting pixlated.